apps/angular/pipe-easy/src/app/shared/pipes/heavy-compute.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { HeavyComputePipe } from './heavy-compute.pipe';
+
+describe('HeavyComputePipe', () => {
+  it('create an instance', () => {
+    const pipe = new HeavyComputePipe();
+    expect(pipe).toBeTruthy();
+  });
+});
```

Comment:
What is the goal of your test ?

Reply:
@tomalaforge this is auto generated i will clean up the unnecessory test files